Formula/fortio.rb

```diff
@@ -0,0 +1,39 @@
+class Fortio < Formula
+  desc "HTTP and gRPC load testing and visualization tool and server"
+  homepage "https://github.com/istio/fortio"
+  url "https://github.com/istio/fortio/archive/v0.9.0.tar.gz"
+  sha256 "76871124e7abd13d7fa0ccc5ad6912bc1c3d77d53694da2b26ad44af3638edab"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["XC_OS"] = "darwin"
+    ENV["XC_ARCH"] = "amd64"
+    ENV["GOPATH"] = buildpath
+    ENV["CGO_ENABLED"] = "0"
+    contents = Dir["{*,.git,.gitignore}"]
+    (buildpath/"src/istio.io/fortio").install contents
+    cd "src/istio.io/fortio" do
+      system "dep", "ensure"
```

Comment:
If we use the Git tag to pull the submodules, is this needed? There's seems to be a vendor directory.

Reply:
In a previous review we have been asked to use the source tarball rather than a git pull.
Unfortunately the tarball doesn't have the content of the `vendor` folder and therefore the use of `dep ensure` to fetch it.